digitalpublicgoods/openrims.json

```diff
@@ -0,0 +1,93 @@
+{
+  "name": "OpenRIMS",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://www.openrims.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/MSH/Pharmadex2"
```

Comment:
The documentation is missing an installation guide. The shared repo only showed the deployment guide. Thanks.